.github/codecov.yml

```diff
@@ -40,6 +40,55 @@ flag_management:
         target: auto # Let's decrease this later.
         threshold: 10
 
+  groups:
+    gno-land:
+      paths:
+        - "gno.land/**"
+      flags:
+        - gno-land
+    gnovm:
+      paths:
+        - "gnovm/**"
+      flags:
+        - gnovm
+    misc:
+      paths:
+        - "misc/**"
+      flags:
+        - misc
+    tm2:
+      paths:
+        - "tm2/**"
+      flags:
+        - tm2
+    contribs:
+      paths:
+        - "contribs/**"
+      flags:
+        - contribs
```

Comment:
Flags are already automatically defined here: https://github.com/gnolang/gno/pull/3316/files#diff-a8d8d916d02d13f09d2a0128a4667158fdd150a4e354d0c7a9cde6a8479b3dc5R68 so no need to define them here.

Reply:
You're right, dropped it:


f5ca5d6